contrib/k8s-lbry/README.md

```diff
@@ -0,0 +1,862 @@
+```
+
+Once in the shell, do whatever maintaince is necessary, and press `Ctrl-D` or
+type `exit` when done.
+
+### k8s-lbry shell-pvc
+
+When doing maintainance tasks, it is often useful to be able to run a utility
+container that mounts the persistent volume
+([PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)) of
+another container. This is especially useful in scenarios where the pod will not
+start, and therefore cannot use the `run.sh shell` command in the previous
+section.
+
+This will run a shell in a new utility container, mounting the lbrynet PVC to
+`/pvcs/lbrynet`.
+
+Example: 
+
+```
+k8s-lbry shell-pvc lbrynet
+```
+
+Once in the shell, do whatever maintaince is necessary, and press `Ctrl-D` or
+type `exit` when done.
+
+
+### k8s-lbry restart
+
+Delete a pod for a given app name. The existing deployment will immediately
+restart a new pod.
+
+Example:
+
+```
+k8s-lbry restart speech
+```
+
+### k8s-lbry lbrynet
+You can run the `lbrynet` client from within the running pod, redirecting output
+to your local console.
+
+Example:
+
+```
+k8s-lbry lbrynet --help
+```
+
+### k8s-lbry chainquery-mysql-client
+
+Run the mysql shell for the chainquery database.
+
+### k8s-lbry speech-mysql-client
+
+Run the mysql shell for the speech database.
+
+### k8s-lbry lbrynet-copy-wallet <local_backup_path>
+
+Backup the lbrynet wallet to a local path.
+
+Example:
+
+```
+k8s-lbry lbrynet-copy-wallet /tmp/k8s-lbry-lbrynet-wallet-backup.json
+```
+### k8s-lbry package
+
+This is for the developer of this package to build and maintain the helm package
+releases, and upload to the S3 package repository. Requires `s3cmd` installed.
+
+Example:
+
+```
+k8s-lbry package 0.1.1
+```
+
+
+## TLS / SSL / HTTPS
+
+You have already installed cert-manager for your cluster, but HTTPs is not
+turned on out of the box. Setup is easy:
+
+ * You need to create a DNS A record for your domain pointing to the External IP
+   address of the nginx-ingress Load Balancer. (Preferably create a wildcard
+   record for an entire subdomain [`*.example.com` or `*.lbry.example.com`],
+   that way you only have to set this up once, no matter how many sub-domains
+   you end up needing.) Refer to the [install
+   section](https://github.com/EnigmaCurry/lbry-docker/tree/k8s-lbry/contrib/k8s-lbry#k8s-lbry-install)
+   for how to retrieve the IP address.
+   
+ * Edit `values-dev.yaml`
+ 
+ * Change `cert-manager-issuer.email` from the example email address to your
+   own. [Let's Encrypt](https://letsencrypt.org/) is a free TLS certificate
+   issuer, and they will send you important emails about your domain and
+   certificate expirations.
+ 
+ * You can turn on the echo service to test with:
+ 
+   * Change `echo-http-server.enabled` to `true`
+   
+   * Change `echo-http-server.hostname` to a hostname you've configured the DNS
+   for.
+ 
+Upgrade nginx-ingress, turning on HTTPs support:
+
+```
+NGINX_ENABLE_HTTPS=true k8s-lbry upgrade-nginx-ingress
```

Comment:
Is this a whole env-var? Looks like a combination between a variable and a command.  This should be clearer.

Reply:
how about this?

```
(
  NGINX_ENABLE_HTTPS=true 
  k8s-lbry upgrade-nginx-ingress
)
```
Only problem with that though is if the user thinks they don't need the parentheses. The env var is only supposed to be temporary for that one command.

Reply:
I'll just leave a note to run it with the parens.